Create branch for review

BregornOriginal opened the pull request:
The objectives of this project are next:

- Build a personal portfolio site
- Use images and backgrounds to enhance the look of the website
- Understand how to parse a Figma design to create a UI
- Flexbox to place elements in the page
In this commit I made the next changes: 

- For all images that I had added, I correct the src of them. So now I don't have Linters problems and it's showing up.
- Evolve the hamburger button whit a nav
- The logo text now is under a "a" tag as is clickable
- Added a correctly "h1" tag to the logo title
- Added a "ul" tag and "li" to icons
- Finally, I fix the white space on the right side of the mobile view

BregornOriginal committed:
Create branch for review

BregornOriginal committed:
Update main.css

Corrected style errors founded by Linters

BregornOriginal committed:
Update index.html

Fixed some errors founded by Linters

BregornOriginal committed:
Move img of icon to folder icon

Added font-family to h2

BregornOriginal committed:
Update main.css

Correct font family of h2

KDlamini commented on index.html:
```diff
@@ -12,9 +12,9 @@
+          src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio\Icons\Enabled.png"
           alt="menu button"
-        >
+        />
```
Let's start with your linter errors that are causing your checks not to pass:

You have the following WebHint errors that have something to do with your `img` tag `src` paths:
![linters](https://user-images.githubusercontent.com/38088818/164545638-c728a771-a141-45ef-a17e-b66cdcbd63f9.png)

For your `hamburger` button:
![image](https://user-images.githubusercontent.com/38088818/164545937-0d92a7d2-fad4-4483-95bf-ea6135c5ee21.png)

The correct syntax is:
```
      <button type="button" class="options-button">
        <img
          class="menu-button"
          src="./Icons/Enabled.png"
          alt="menu button"
        />
      </button>
```

This also solves your icon not displaying issue as well as seen below:
![image](https://user-images.githubusercontent.com/38088818/164546308-7cb32943-c4fb-4426-98c2-7f284332645e.png)

KDlamini commented on index.html:
```diff
@@ -33,27 +33,27 @@ <h3>I'am a Software Developer</h3>
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Microverse Icon.png"
           alt="microverse icon"
-        >
+        />
```
For your social icons your `src` paths should be like this:
```
<div class="container-icons">
        <img
          class="main-icons"
          src="./Icons/Git-hub-icon.png"
          alt="git-hub icon"
        />
        <img
          class="main-icons"
          src="./Icons/Icon - Linkedin.png"
          alt="instagram icon"
        />
        <img
          class="main-icons"
          src="./Icons/Icont number 3.png"
          alt="new icon"
        />
        <img
          class="main-icons"
          src="./Icons/Twitter icon.png"
          alt="twitter icon"
        />
        <img
          class="main-icons"
          src="./Icons/Microverse Icon.png"
          alt="microverse icon"
        />
      </div>
```

As well as your illustration container image:
```
    <div class="container-illustration">
      <img
        class="poligon-illustration"
        src="./Icons/Illustration.png"
        alt="Image of poligons"
      />
    </div>
```

Now your linter issues are solved and also your images are also displaying properly:
Before | After
------------- | -------------
![image](https://user-images.githubusercontent.com/38088818/164547920-9b4060bd-e086-4517-934c-885d7bae858c.png) | ![image](https://user-images.githubusercontent.com/38088818/164547812-4b055a7e-d700-4da2-89c1-864880768827.png)

KDlamini replied:
The list of icons should use a `ul` tag to group them, and `li` per icon. Each icon should be a link (`a` tag) because it must be clickable in order to redirect you to the clicked social media website.

The structure should look more or less like like:
```
      <ul class="container-icons">
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Git-hub-icon.png"
              alt="git-hub icon"
            />
          </a>
        </li>
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Icon - Linkedin.png"
              alt="instagram icon"
            />
          </a>
        </li>
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Icont number 3.png"
              alt="new icon"
            />
          </a>
        </li>
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Twitter icon.png"
              alt="twitter icon"
            />
          </a>
        </li>
        <li>
          <a href="#">
            <img
              class="main-icons"
              src="./Icons/Microverse Icon.png"
              alt="microverse icon"
            />
          </a>
        </li>
      </ul>
```

KDlamini commented on index.html:
```diff
@@ -12,9 +12,9 @@
+        />
       </button>
       <p class="ptitle">Julio's Porfolio</p>
     </header>
```
Now that we are done with the linters that have been a nightmare for us. Let's solve some technical issues that do not quite hit the project requirements.

The `hamburger` button should be inside a `nav` element and the `logo text` should be clickable. Therefore, the logo text should be an `<a>` tag. You should have something that looks like this:
![image](https://user-images.githubusercontent.com/38088818/164548984-7a2d4449-710d-47a1-94ea-968b40c68c47.png)

KDlamini commented on index.html:
```diff
@@ -33,27 +33,27 @@ <h3>I'am a Software Developer</h3>
           src="G:\Julio\Programación\Microverse\Repositories\Julio-Gagliardi-Portfolio-\Icons\Git-hub-icon.png"
           alt="git-hub icon"
-        >
+        />
```
![image](https://user-images.githubusercontent.com/38088818/164549334-3f9d5dbc-03ae-4340-b41c-79f7ac468b85.png)
Line 23: should be an `h1` tag since it is the largest text on the page and according to HTML guidelines, a page should have an `h1` tag.

KDlamini commented on main.css:
```diff
@@ -31,99 +34,87 @@
   padding: 8px;
   border: none;
 }
+
```
Last but not least, kindly fix the white space on the right side of your mobile view:

![image](https://user-images.githubusercontent.com/38088818/164551128-6651abe6-eade-48a7-9285-ab5ffe564afa.png)

KDlamini reviewed (changes requested):
### Hi @BregornOriginal 🙋‍♀️,  

Great job so far! 
You have done a great job creating a mobile skeleton according to the Figma design. 
 There are some issues that you still need to work on improving your project according to the requirements. You are almost there! 💪🏾 

## Required Changes ♻️
 Normally, I should be invalidating your pull request because your linters are not passing, you do not have the correct title, and you do not have a PR description. However, I noticed that you were having difficulties with your linters and you've deleted about four branches already. Let's diagnose the problem together.

 - Please use the correct Pull request title. Your title should be something like "setup and mobile version skeleton" since this is what this milestone is about. The good news is that you can still edit the title 💪🏾 
 - Please write a proper Pull Request description about the steps you took to set up the toolbar and the headlines. The simplest way to do this is to list the project requirements that are on the project activity. The good news is that you can still edit the description, it's not too late 💪🏾 
 - Please name your branches properly in the future. Your branch name should be descriptive of the project you are doing i.e `mobile-setup` or `mobile-skeleton` instead of `pr-for-review`. However, there's nothing we can do to change that now, but please consider naming your branches properly in the future.
 
**Kindly look through the comments below 👇 for required changes**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers 🥂  and Happy coding!!! 👯 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @KDlamini  in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

BregornOriginal committed:
setup and mobile version skeleton

The objetives of this project are next
- Build a personal portfolio site
- Use images and backgrounds to enchance the look of the website
- Understand how to parse a Figma design to create a UI
- Flexbox to place elements in the page
In this commit i made the next changes:

- For all images that I had added, I correct the src of them. So now I don't have Linters problems and it's show up.
- Evolve the hamburger button whit a nav
- The logo text now is under a "a" tag as is clickeable
- Added a correctly "h1" tag to the logo title
- Added a "ul" tag and "li" to icons
- Finaly I fix the white space on the right side of mobile view

BregornOriginal committed:
Name of files were corrected

- I have deleted spaced in the name of files

BregornOriginal committed:
Update index

Spaces between words in the src were deleted.

BregornOriginal committed:
Missed a space in a file. I Delete it

BregornOriginal committed:
Added node_modules to .gitignore

- Also eliminate dots from "li"tags

Lucas-Masaba reviewed (approved):
# Hello @BregornOriginal  👋 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🍔  
Congratulations! 🎉 

![almost-there](https://media.giphy.com/media/BPJmthQ3YRwD6QqcVD/giphy.gif)

## Highlights 🍾 
- Well written README 😎 
- Well written pull request description 🏄 
- Linter checks are passing 👍 

## Optional suggestions 💭 

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏 💻 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

[OPTIONAL]
- Please ensure to remove the white space as seen below 👀 . This will help you in future milestones as you build your portfolio 👍 
![image](https://user-images.githubusercontent.com/45460192/164569076-7abbacb1-8c00-42f9-bd5a-4aad7cd09fd1.png)

BregornOriginal commented:
Hi @KDlamini and @Lucas-Masaba . Thanks all for your reviews. They were very useful to me because I learn new things today whit this project. And yes! After I did the request review I remember about that white space, and correct this! 
Best for both!
Julio.

Lucas-Masaba commented:
🏂We are always here to help. I hope you have an awesome day🕺
